compiler/showTypes.bal

```diff
@@ -39,4 +39,45 @@ function subtypeRels(front:SourcePart[] sources) returns string[]|err:Any|io:Err
         order by s
         select s;
     return rels;
-}
\ No newline at end of file
+}
+
+function testSubtypes(front:SourcePart[] sources, string[] expected) returns boolean|err:Any|io:Error {
+    var [env, m] = check front:typesFromString(sources);
+    var tc = t:typeContext(env);
+    foreach var item in expected {
+        int? i1 = item.indexOf(" < ");
+        if i1 is int {
```

Comment:
Lot of logic in these three `if`s seems to be repeating, please deduplicate via functions.